contributor/02-becoming-a-contributor.md

```diff
@@ -0,0 +1,45 @@
+# Draft comments
+
+Outline for this section
+
+    What does it take to be successful as an InnerSource contributor? This segment shares information on how contributors to InnerSource projects think and act differently from team members in other projects. It highlights which types of contributions are seen as valuable InnerSource contributions.
+
+## Article
+
+### Becoming an InnerSource Contributor
+
+InnerSource contributors operate outside of regular team boundaries, they are the links crossing silos within organisations. As such they need to be aware of a few common practices that make this work more effective.
+
+#### Sharing Mindset
```

Comment:
We'll need to remember to fix the header levels at the end. They are verbatim translated into O'Reilly sections and subsections...
Goes for all articles in the project...

Reply:
What is the desired header level?

Reply:
http://docs.atlas.oreilly.com/writing_in_markdown.html#writing-markdown-for-htmlbook-Oytyf4

Reply:
Let's capture in issue before merge of this article.

Reply:
Added issue #114

Reply:
Fantastic!